uwblueprint/abtc
# Add Admin: Accept or Reject Page

## Notion ticket link

[Admin: Accept or Reject Page](https://www.notion.so/uwblueprintexecs/Admin-Accept-or-Reject-Page-34d3f574d52c4e5cadcb306efcb20959?pvs=4)



## Implementation description
* added accept and reject page UI
* added basic functionality -> clicking top check mark selects all and pagination (current only displaying 3 per page)
* for the other functionality (accept/reject) -> added console.log
* used mock data for now-> need to use get platformSignupRequests endpoint



## Steps to test
1. go to [http://localhost:3000/platform-signup-requests]( http://localhost:3000/platform-signup-requests)
2. check pagination 



## What should reviewers focus on?
* UI matches figma 


## Checklist

## Screenshot
<img width="1428" alt="Screenshot 2024-06-11 at 7 30 12 PM" src="https://github.com/uwblueprint/abtc/assets/65462473/f6178195-53db-4ab8-ac37-b8999bed33ea">
